functions/source/c1c_controltower_lifecycle.py

```diff
@@ -375,8 +375,6 @@ def lambda_handler(event, context):
                 update_policy(event["account_id"])
             elif event["InvokeAction"] == "remove_account_config":
                 remove_account_config(event["account_id"])
-            elif event["InvokeAction"] == "remove_all":
-                remove_all(context.function_name)
```

Comment:
@XeniaP what was the purpose of removing this option?

Reply:
Hi @glb, this is a correction of my last commit, in my test this function only delete the aws resources but don't delete the accounts in Cloud One.